packages/fx-core/src/common/permissionInterface.ts

```diff
@@ -8,6 +8,30 @@ export interface ResourcePermission {
   roles: string[] | undefined;
 }
 
+export enum CollaborationState {
+  OK,
+  NotProvisioned,
+  M365AccountNotMatch,
```

Comment:
Not match seems confusing. What do you mean by not match here?

Reply:
User login a different m365 account that is not belong to current project

Reply:
You mean whether the account has the permission? Then sth. like NoPermission may be better?

Reply:
it is not just no permission, it is M365 account in different tenant

Reply:
That's OK, and maybe we can consider about a better naming here.

Reply:
Can we re-name it to show the tenant not match?

Reply:
Sure, will update